### PR TITLE
Frans

### PR DESCRIPTION
**Pull request by fransbianchi**
issue 94 done

### CODE REVIEW

File: R/create_map.R [hunk 1]
```diff
@@ -16,7 +23,6 @@ create_map <- function(png_filename, show_bridge_openings = TRUE) {
     ggmap::get_map(bbox, maptype = "toner_stamen", quiet = TRUE))
   p <- ggmap::ggmap(groningen)
   if (show_bridge_openings) {
-    data <- informalr::get_bridge_openings()
```
Comment (richelbilderbeek) [inline R/create_map.R:19]: Deleting this line seems like an error to me. I suggest to put it back first :grin: :+1:

**Review by richelbilderbeek — CHANGES_REQUESTED** [review]
Brilliant works, except for an accidentally deleted line. I suggest you add the line again, then resubmit :+1:

**Comment by fransbianchi** [comment]
I have fixed the issue added the deleted line and the param on top

**Review by richelbilderbeek — APPROVED** [review]
Brilliant work :+1:

**Pull request merged**